### PR TITLE
added gjg fun fact

### PR DESCRIPTION
**Pull request by evolshenanigans**
added my fun fact

### CODE REVIEW

**Review by ajmompr — APPROVED** [review]
Looks great!

File: README.md [hunk 1]
```diff
@@ -83,3 +83,4 @@ Clone this repository (or open in codespace), create a branch with your name, ad
 
 | Name                 | Fun Fact |
 | -------------------- | --- |
+| Gustavo Gutierrez.   | The reason Tarzan was clean shaven even though he was raised in the jungle because in the novel he found his dads shaving kit and used it growing up. The movie doesnt show that part. |
```
Comment (heratyian) [inline README.md:86]: maybe "is because"?

**Review by heratyian — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (evolshenanigans) [reply README.md]: fixed grammar

**Review by heratyian — APPROVED** [review]
👍

**Review by ajmompr — APPROVED** [review]
Looks good; resolve new conflicts and merge

**Pull request merged**